wp-content/plugins/hurumap/src/FlourishBlock/Save.js

```diff
@@ -5,14 +5,15 @@ function Save({ attributes }) {
   return (
     <div
       id={`indicator-flourish-${attributes.chartId}`}
+      style={{ width: attributes.chartWidth || '100%' }}
       data-chart-id={attributes.chartId}
       data-chart-title={attributes.title}
       data-chart-description={attributes.description}
       data-show-insight={attributes.showInsight}
       data-insight-title={attributes.insightTitle}
       data-insight-summary={attributes.insightSummary}
       data-data-link-title={attributes.dataLinkTitle}
-      data-analysis-link-href={`/profiles/${attributes.analysisCountry}`}
+      data-analysis-country={attributes.analysisCountry}
```

Comment:
Can we be a bit more explicit here (`country-slug` or `country-code`)

Reply:
@KhadijaMahanga